src/components/Timestamp/Timestamp.tsx

```diff
@@ -28,6 +28,16 @@ const timeStyles = css`
 	white-space: nowrap;
 `;
 
+const anchorStyles = css`
+	display: none;
+	width: 15px;
+	height: 15px;
+	vertical-align: middle;
+	margin-left: 0.125rem;
+	max-height: 0.9375rem;
+	fill: #ffffff;
+`;
```

Comment:
```suggestion
const anchorStyles = css`
	display: none;
	width: 15px;
	height: 15px;
	vertical-align: middle;
	margin-left: 0.125rem;
	max-height: 0.9375rem;
	fill: #ffffff;
	&:hover {
		display: inline-block;
	}
`;
```

Would this work in response to [your comment](https://github.com/guardian/discussion-rendering/pull/550/files#r789783418)?

This might might negate the need to add the CSS in [two places](https://github.com/guardian/discussion-rendering/pull/550/files#diff-726f6a4e18a031a44d7ef6e6019e0bbf742d7755f4015edf9c6e955839efdefaR160).

Reply:
Gave it a go, afraid it didn't work. The problem is the link icon needs to be shown when any part of the comment is hovered over - not just the timestamp - which is why I needed to apply some CSS styling in the parent Comment class to capture that hover